app/routers/chat_stream.py

```
@@ -139,37 +213,178 @@ async def _generate_chat_stream(request: ChatRequestModel):
+            "Access-Control-Allow-Origin": "*",
+        }
+
+        return StreamingResponse(
+            _generate_chat_stream(request, db_session, chat_session.id, user_message.id),
+            media_type="text/event-stream",
+            headers=headers
+        )
+
+    except Exception as e:
+        logger.error(f"❌ 채팅 스트림 초기화 실패: {e}")
+        raise HTTPException(status_code=500, detail=f"채팅 스트림 초기화 실패: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)